sql/dbschema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE users (
+  userId INT PRIMARY KEY IDENTITY(1,1),
+  firstName VARCHAR(50) NOT NULL,
+  lastName VARCHAR(50) NOT NULL,
+  email VARCHAR(50) NOT NULL,
+  deleted INT DEFAULT 0 NOT NULL
+);
+
+CREATE TABLE tasks(
+  taskId INT PRIMARY KEY IDENTITY(1,1),
+  userId INT NOT NULL,
+  title VARCHAR(50),
+  description VARCHAR(400),
+  createdTime DATETIME NOT NULL,
```

Comment:
Do we also want to include when the task was completed?

Reply:
Further, do we want to include a due date for tasks? That seems like it'd be pretty useful. I know that having a perfect task management app is not exactly the point of this project, but I still think it might be worth thinking about?

Reply:
good ideas, I will add those.

Reply:
I think this is a terrific idea for maybe next sprint, but not for sprint 1 with the time we have left. For example, as I am writing the script to "update" a task, I am currently focusing on the existing fields, and would even forgo the create timestamp at this point of time (scope creep).